content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -363,3 +364,5 @@ this problem.
 
 For guidance on administering DNS configurations, check
 [Configure DNS Service](/docs/tasks/administer-cluster/dns-custom-nameservers/)
+
+[CoreDNS]: /docs/tasks/administer-cluster/coredns/
```

Comment:
I'd link to https://coredns.io/ rather than the task page.

Reply:
done